drakrun/drakrun/main.py

```diff
@@ -400,13 +400,18 @@ def process(self):
                 full_cmd = cur_start_command
                 self.log.info("Using command: %s", full_cmd)
 
-                drakvuf_cmd = ["drakvuf",
-                               "-o", "json",
-                               "-x", "poolmon",
-                               "-x", "objmon",
-                               "-x", "socketmon",
-                               "-x", "dkommon",
-                               "-x", "envmon",
+                task_quality = self.current_task.headers.get("quality", "high")
+                excl_str = self.config.config['drakrun'].get('excluded_plugins', 'poolmon, objmon, socketmon, dkommon, envmon')
+                excl_q_str = self.config.config['drakrun'].get(f'excluded_plugins_quality_{task_quality}', '')
+                excl_cmdline = []
+
```

Comment:
IMO, this should be moved into initialization step and queried like: `self.disabled_plugins[quality]`.

Reply:
olk